src/utils/log-test.js

```diff
@@ -31,15 +29,15 @@ describes.realWin('warn', () => {
 
   it('should log a warning', () => {
     warn('Hello World');
-    expect(console.warn.calledWith('Hello World')).to.be.true;
+    expect(self.console.warn.calledWith('Hello World')).to.be.true;
```

Comment:
OOC can this be written `.to.be.calledWith(...)`? .to.be.true seems redundant.

Reply:
Good call, updated